jme3-core/src/main/java/com/jme3/animation/SkeletonControl.java

```diff
@@ -76,6 +75,11 @@
      */
     private SafeArrayList<Mesh> targets = new SafeArrayList<Mesh>(Mesh.class);
     /**
+     * Geometry with an animated mesh, for calculating attachments node
+     * transforms. A null means no geometry is subject to this control.
+     */
+    private Geometry targetGeometry = null;
```

Comment:
mhh wait... I did,'t realize but targets is actually a Mesh list not a Geometry list....
Maybe we could change this and just call geom.getMesh()